console-extensions.json

```diff
@@ -48,6 +48,13 @@
       "component": { "$codeRef": "PolicyTopologyPage" }
     }
   },
+  {
+     "type": "console.page/route",
+     "properties": {
+       "path": "/kuadrant/authpolicy/new",
```

Comment:
I'm not convinced we need custom routes for any that we 'override'... yet.
All links to create a new auth policy should render the component referenced further below

Reply:
removed